ironfish/src/wallet/walletdb/transactionValue.ts

```diff
@@ -41,6 +44,14 @@ export class TransactionValueEncoding implements IDatabaseEncoding<TransactionVa
       bw.writeU32(sequence)
     }
 
+    const assetCount = value.assetBalanceDeltas.size
+    bw.writeU32(assetCount)
+
+    for (const [assetIdentifier, balanceDelta] of value.assetBalanceDeltas.entries()) {
```

Comment:
the default iterator for buffer set is entries, so you don't need the `.entries()` call here